From 90d7a9c15f2d6014b5d34375c06aea1015e83907 Mon Sep 17 00:00:00 2001
From: zhang sanshan <b51434@freescale.com>
Date: Thu, 5 Nov 2015 12:49:26 +0800
Subject: [PATCH 7/7] MA-7218-1 [Android_imx6_auto]factoryreset:do factory
 reset with binay ubiupdatevol in recovery

The lib used by binary must be static.
ubiupdatevol used in recovery,so it also must be static.

Signed-off-by: zhang sanshan <b51434@freescale.com>
---
 Android.mk | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/Android.mk b/Android.mk
index 46e8118..306e8e8 100644
--- a/Android.mk
+++ b/Android.mk
@@ -222,9 +222,11 @@ LOCAL_C_INCLUDES += $(LOCAL_PATH)/ubi-utils/include
 LOCAL_C_INCLUDES += $(LOCAL_PATH)/include
 LOCAL_C_INCLUDES += $(LOCAL_PATH)/ubi-utils/src
 
-LOCAL_SHARED_LIBRARIES := libubi
+LOCAL_STATIC_LIBRARIES := libubi libc
 
 LOCAL_MODULE := ubiupdatevol
+LOCAL_FORCE_STATIC_EXECUTABLE := true
+LOCAL_MODULE_PATH := $(TARGET_ROOT_OUT_SBIN)
 LOCAL_MODULE_TAGS := optional
 
 include $(BUILD_EXECUTABLE)
-- 
1.8.0

